sdk/azidentity/TROUBLESHOOTING.md

```diff
@@ -21,6 +20,7 @@ This troubleshooting guide covers failure investigation techniques, common error
   - [Azure Virtual Machine managed identity](#azure-virtual-machine-managed-identity)
 - [Troubleshoot UsernamePasswordCredential authentication issues](#troubleshoot-usernamepasswordcredential-authentication-issues)
 - [Troubleshoot WorkloadIdentityCredential authentication issues](#troubleshoot-workloadidentitycredential-authentication-issues)
+- [Troubleshoot AzurePipelinesCredential authentication issues](#troubleshoot-azurepipelinescredential-authentication-issues)
```

Comment:
Can you please revert this? The ToC is alphabetized for easy scanning and doesn't need to follow the section ordering below.

Reply:
Oh, interesting. I didn't realize the intent was to keep the ToC in alphabetical order.

Reply:
I doubt it matters because readers typically know the one section they want to read and arrive either at that section itself because they followed a link from an error message, or at the top of the doc i.e. the ToC.